language_detector/main.py

```diff
@@ -1,9 +1,16 @@
 """This is the entry point of the program."""
-
+# coding=utf-8
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    spanish, german, english = [set(LANGUAGES[x]['common_words']) for x in range(3)]
+    language_list = [spanish, german, english]
```

Comment:
To add a bit more detail to my overall comment, this is where I was talking about hard coding the language names. I think instead of specifying each language you could have just used a for loop to get the common words and language names. That way you could handle these 3 languages or 100 languages.